virttest/utils_misc.py

```diff
@@ -2757,7 +2757,8 @@ def get_winutils_vol(session, label="WIN_UTILS"):
 
     :return: volume ID.
     """
-    return get_win_disk_vol(session, condition="VolumeName='%s'" % label)
+    return wait_for(lambda: get_win_disk_vol(session,
+                    condition="VolumeName='%s'" % label), 30, 0, 1)
```

Comment:
Here the timeout value is 30s, and it is less than the one (120s) in callee (`get_win_disk_vol`), so that is incorrect.

Reply:
oo, you are right, I missed the timeout value in get_win_disk_vol func, I update the wait for timeout to 240.